tests/arduino_lib/Makefile

```diff
@@ -0,0 +1,9 @@
+include ../Makefile.tests_common
+
+BOARD_INSUFFICIENT_MEMORY := arduino-duemilanove arduino-leonardo arduino-nano \
+                             arduino-uno nucleo-f031k6
+
+USEPKG += talking_leds
```

Comment:
For simplicity, the talking_leds package should have a dependency to the arduino_lib module. Just add a Makefile.dep file containing `USEMODULE += arduino_lib` in the `pkg/talking_leds` directory.
The line below becomes useless then.

Reply:
Fixed.